lib/auth/auth_with_roles.go

```diff
@@ -145,6 +145,26 @@ func (a *ServerWithRoles) authConnectorAction(resource string, verb string) erro
 	return nil
 }
 
+// identityCenterAction wraps [ServerWithRoles.identityCenterActionNamespace],
+// supplying the default namespace for convenience.
+func (a *ServerWithRoles) identityCenterAction(resource string, verbs ...string) error {
+	return a.identityCenterActionNamespace(apidefaults.Namespace, resource, verbs...)
+}
+
+// identityCenterActionNamespace is a special checker that grants access to
+// Identity Center resources. In order to simplify the writing of role condition
+// statements, the various Identity Center resources are bundled up under an
+// umbrella `KindIdentityCenter` resource kind. This means that if access to the
+// target resource is not explicitly denied, then the user has a second chance
+// to get access via the generic resource kind.
+func (a *ServerWithRoles) identityCenterActionNamespace(namespace string, resource string, verbs ...string) error {
```

Comment:
Nit: We literally never use a namespace other than "default", do we really need a separate `identityCenterActionNamespace` method?

I would consider just keeping `identityCenterAction` and hardcoding `apidefaults.Namespace` to keep things a little simpler.